xtask/src/workload.rs

```diff
@@ -51,38 +29,38 @@ pub fn parse(
             percentage_cold_transfer: percentage_cold_transfer.unwrap_or(0),
             additional_initial_capacity,
         }),
-        "set_balance" => Box::new(CustomWorkload {
-            reads: 0,
-            writes: 100,
-            delete: 0,
-            update: 0,
+        "set_balance" | "heavy_write" => Box::new(CustomWorkload::new(
```

Comment:
why are these the same?

Reply:
A week ago, during a discussion, it emerged that the two main workloads could have been transfer and set_balance, but set_balance is just a bunch of writes into the storage (if I'm not mistaken). Thus, having two separate things, set_balance and the custom workload, seems useless, so I just named set_balance as the default workload.

Anyway, the default workloads will likely become easier, such as `randr`, `randw`, etc., and so those names will disappear